source/crud/query/specify-documents-to-return.txt

```diff
@@ -23,6 +23,303 @@ Overview
+     { "_id": 2, "name": "Les Misérables", "author": "Hugo", "length": 1462 },
+     { "_id": 3, "name": "Atlas Shrugged", "author": "Rand", "length": 1088 },
+     { "_id": 4, "name": "Infinite Jest", "author": "Wallace", "length": 1104 },
+     { "_id": 5, "name": "Cryptonomicon", "author": "Stephenson", "length": 918 },
+     { "_id": 6, "name": "A Dance With Dragons", "author": "Martin", "length": 1104 },
+   ]);
+
+.. include:: /includes/access-cursor-note.rst
+
+.. _node-fundamentals-limit:
+
+Limit
+-----
+
+Use the ``limit()`` method to cap the number of documents that can be returned from a read
+operation. This method specifies the maximum number of documents that the operation can
+return, but the operation can return a smaller number of documents if there are not enough
+documents present to reach the limit. If ``limit()`` is used with the :ref:`skip()
+<node-fundamentals-skip>` method, the skip applies first and the limit only applies to the
+documents left over after the skip.
+
+The following example queries the collection to return the top three
+longest books. 
+
+- The query matches all documents because the query filter is empty.
+- The descending ``sort()`` method on the ``length`` field returns longer books before shorter books.
+- The ``limit()`` method specifies returning only the first ``3`` results.
+
+.. code-block:: javascript
+   :emphasize-lines: 4
+
+   // define an empty query document
+   const query = {};
+   // sort in descending (-1) order by length
+   const sort = { length: -1 };
+   const limit = 3;
+   const cursor = myColl.find(query).sort(sort).limit(limit);
+   for await (const doc of cursor) {
+       console.dir;
+   }
+
+The code example above outputs the following three documents, sorted by
+length:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "title": "Les Misérables", "author": "Hugo", "length": 1462 }
+   { "_id": 6, "title": "A Dance With Dragons", "author": "Martin", "length": 1104 }
+   { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
+
+.. note::
+
+   The order in which you call ``limit()`` and ``sort()`` does not matter because the
+   driver reorders the calls to apply the sort first. The following two calls are
+   equivalent:
+
+   .. code-block:: javascript
+
+      myColl.find(query).sort({ length: -1 }).limit(3);
+      myColl.find(query).limit(3).sort({ length: -1 });
+
+You can also apply ``sort()`` and ``limit()`` by specifying them in an
+``options`` object in your call to the ``find()`` method. The following two
+calls are equivalent:
+
+.. code-block:: javascript
+
+   myColl.find(query).sort({ length: -1 }).limit(3);
+   myColl.find(query, { sort: { length: -1 }, limit: 3 });
+
+For more information on the ``options`` settings for the ``find()``
+method, see the
+`API documentation on find() <{+api+}/classes/Collection.html#find>`__.
+
+.. _node-fundamentals-sort:
+
+Sort
+----
```

Comment:
Okay I actually think the Sort section should come first (before both skip and limit) since it's used in both of them, should've clarified that last time oops